proto/p4/config/p4info.proto

```diff
@@ -116,6 +116,8 @@ message Table {
   repeated uint32 direct_resource_ids = 7;
   int64 size = 8;  // max number of entries in table
   bool with_entry_timeout = 9;  // entry ageing is enabled for table
+  // table with static P4 entries, cannot be modified at runtime
```

Comment:
LGTM for this change.
What about reading const tables? Should we have a mechanism to filter out const tables while doing a wildcard read from the controller? This would be a useful feature in p4runtime for reconciliation.

Reply:
I believe a wildcard Read RPC on all tables should return the contents of const tables as well. Maybe the controller can rely on the P4Info message - with this new proposed attribute - to figure out which tables are const and filter out the entries itself?

Reply:
I'm going to merge this PR so I can make progress on https://github.com/p4lang/PI/issues/295
The issue brought up by @samar-abdi can be discussed at the P4 API WG meeting if time or in a separate Github issue.